task002/fibonacci/fibonacci.go

```diff
@@ -0,0 +1,19 @@
+package fibonacci
+
+import "fmt"
+
+func fibonacci() func() int {
+	i, j := 0, 1
+	return func() int {
+		nextFibonacci := i + j
```

Comment:
It is common in Go to use short variable names. Variable name 'next' would be more readable then 'nextFibonacci'.